code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -233,6 +233,9 @@
 	new /obj/item/clothing/neck/tie/red(src)
 	new	/obj/item/clothing/neck/tie/black(src)
 	new /obj/item/clothing/neck/tie/detective(src)
+	new /obj/item/clothing/under/ship/kiryujumpsuit(src)

```

Comment:
```suggestion
	new /obj/item/clothing/under/ship/kiryujumpsuit(src) //NSV13 - Alternative detective suit

```

When we change files outside of our folder, we mark them loosely with a comment so that we don't accidentally lose them whenever we upstream merge.